package.json

```diff
@@ -31,28 +31,28 @@
         "test": "jest"
     },
     "dependencies": {
-        "@lwc/compiler": "2.2.3",
-        "@lwc/engine-dom": "2.2.3",
+        "@lwc/compiler": "2.2.6",
+        "@lwc/engine-dom": "2.2.6",
         "@lwc/jest-preset": "11.0.1",
         "@lwc/jest-resolver": "11.0.1",
         "@lwc/jest-serializer": "11.0.1",
         "@lwc/jest-transformer": "11.0.1",
-        "@lwc/module-resolver": "2.2.3",
-        "@lwc/synthetic-shadow": "2.2.3",
-        "@lwc/wire-service": "2.2.3",
+        "@lwc/module-resolver": "2.2.6",
+        "@lwc/synthetic-shadow": "2.2.6",
+        "@lwc/wire-service": "2.2.6",
         "@salesforce/wire-service-jest-util": "4.0.0",
         "chalk": "~4.1.1",
-        "fast-glob": "^3.2.5",
-        "jest": "26.6.3",
+        "fast-glob": "^3.2.7",
+        "jest": "27.0.6",
         "yargs": "~17.0.1"
     },
     "devDependencies": {
         "babel-eslint": "^10.1.0",
-        "eslint": "^7.28.0",
-        "husky": "^5.2.0",
+        "eslint": "^7.30.0",
+        "husky": "^7.0.1",
```

Comment:
You can now remove https://github.com/salesforce/sfdx-lwc-jest/blob/master/.husky/.gitignore